scenes/Restaurant/OutlineSprite.gd

```diff
@@ -30,5 +30,6 @@ func _ready() -> void:
 	_update_parent_field()
 
 func _process(_delta: float) -> void:
-	position = _parent.position
-	scale = _parent.scale
+	if _parent != null:
+		position = _parent.position
+		scale = _parent.scale
```

Comment:
This change is unrelated to this 'panning to a new customer' issue, but it's valuable. Make sure to document it in the commit comment

Reply:
Commit comment now mentions "NPE when OutlineSprite's parent was undefined".